electroncash_gui/qt/multi_transactions_dialog.py

```diff
@@ -0,0 +1,242 @@
+        self.save_button.setText("Save")
+        self.save_button.setEnabled(True)
+        self.sign_button.setEnabled(can_sign)
+        self.broadcast_button.setEnabled(self.are_transactions_complete())
+
+        self.num_tx_label.setText(f"Number of transactions: <b>{len(transactions)}</b>")
+
+        sats_per_unit = 10 ** BCH_decimals
+        sum_in_value, sum_out_value, sum_fees = 0, 0, 0
+        self.transactions_table.setRowCount(len(transactions))
+        self.transactions_table.setHorizontalHeaderLabels(self._horiz_header_labels)
+        has_missing_input_values = False
+        for i, tx in enumerate(transactions):
+            out_value = tx.output_value()
+            sum_out_value += out_value
+            try:
+                in_value = tx.input_value()
+            except transaction.InputValueMissing:
+                has_missing_input_values = True
+                fee_item = QtWidgets.QTableWidgetItem("N.A.")
+                fee_item.setToolTip(
+                    "Raw signed transactions don't specify input amounts"
+                )
+                # TODO: asynchronously fetch the input values from the network to
+                #       update the item and labels without slowing down the user
+            else:
+                fee = in_value - out_value
+                sum_in_value += in_value
+                sum_fees += fee
+                fee_item = QtWidgets.QTableWidgetItem(f"{fee / sats_per_unit:.8f}")
+
+            self.transactions_table.setItem(
+                i, 0, QtWidgets.QTableWidgetItem(f"{len(tx.inputs())}")
+            )
+            self.transactions_table.setItem(
+                i, 1, QtWidgets.QTableWidgetItem(f"{len(tx.outputs())}")
+            )
+            self.transactions_table.setItem(
+                i, 2, QtWidgets.QTableWidgetItem(f"{out_value / sats_per_unit:.8f}")
+            )
+            self.transactions_table.setItem(i, 3, fee_item)
+
+            # Print the output addresses on colored background, with a color depending
+            # on the hash of the output addresses. This helps with controlling that
+            # all the outputs are the same, when needed.
+            addresses_set = {addr.to_cashaddr() for (_, addr, _) in tx.outputs()}
+            addresses_txt = ", ".join(sorted(addresses_set))
+            color_item = QtWidgets.QTableWidgetItem(addresses_txt)
+            color_item.setToolTip(addresses_txt)
+            h = sha256(addresses_txt.encode("utf8"))
+            color_item.setBackground(QtGui.QColor(h[0], h[1], h[2]))
+            self.transactions_table.setItem(i, 4, color_item)
+
+        self.out_value_label.setText(
+            f"Total output value: <b>{sum_out_value / sats_per_unit:.8f} {unit}</b>"
+        )
+        if not has_missing_input_values:
+            self.in_value_label.setText(
+            f"Total input value: <b>{sum_in_value / sats_per_unit:.8f} {unit}</b>"
```

Comment:
This line should be indented.  This can probably cause a RuntimeError (to be tested).

Reply:
Done in: https://github.com/scinklja/Electron-Cash/commit/62d235308cd49dcebb62937e2e223149fa05965c

(Didn't seem to cause Runtime Error.)